src/Microsoft.DotnetOrg.Policies/Rules/PR18_ReposShouldNotUseDeprecatedBranchNames.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+
+namespace Microsoft.DotnetOrg.Policies.Rules
+{
+    internal sealed class PR18_ReposShouldNotUseDeprecatedBranchNames : PolicyRule
+    {
+        public override PolicyDescriptor Descriptor { get; } = new PolicyDescriptor(
+            "PR18",
+            "Repos shouldn't use deprecated branch names",
+            PolicySeverity.Hidden
+        );
+
+        public override void GetViolations(PolicyAnalysisContext context)
+        {
+            var deprecatedBranchNames = new (string Deprecated, string Preferred)[]
+            {
+                ("master", "main")
+            };
+
+            foreach (var repo in context.Org.Repos)
+            {
+                if (repo.IsArchived || repo.IsSoftArchived())
+                    continue;
+
+                if (repo.IsFork)
+                    continue;
+
+                if (!repo.IsOwnedByMicrosoft())
+                    continue;
```

Comment:
Think we should be validating that the default branch of the repository is `main` as that is the goal we are going for. The API seems to return a `"default_branch"` item that we could leverage

Reply:
That's problematic because the docs generally don't use main as their default branch but `live` (they also have a `main` branch though).